### PR TITLE
PS-7689 Release Notes for 5.7.34-37

### PR DESCRIPTION
**Pull request by patrickbirch**
Changes to be committed:
	modified:   source/conf.py
	new file:   source/release-notes/Percona-Server-5.7.34-37.rst
	modified:   source/release-notes/release-notes_index.rst

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.7.34-37.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+in addition to enterprise-grade features developed by Percona.
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-4497`: Incorrect option error message for mysqlbinlog
+* :jirabug:`PS-7498`: Prevent the replication coordinator thread stuck in Waiting until MASTER_DELAY seconds after master executed event while handling partial relay log transactions.
+* :jirabug:`PS-7566`: Correct version matching in RPM spec changelog for PS packages
+* :jirabug:`PS-7574`: Missing xml file with MTR test results on Jenkins
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.7.34-37.rst]: Internal task only

File: doc/source/release-notes/Percona-Server-5.7.34-37.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* :jirabug:`PS-7593`: When changing the tx-isolation on a session, after a transaction has executed, the change is not honored and may cause a service failure.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.7.34-37.rst:21]: Missing PS-7578 : Replication failure with UPDATE when replica server has a PK and source not

File: doc/source/release-notes/Percona-Server-5.7.34-37.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* :jirabug:`PS-7636`: Fix CPU features detection in MyRocks
+* :jirabug:`PS-7651`: Travis CI and Azure: add clang-12 testing
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-5.7.34-37.rst]: PS-7651 and PS-7690 are Admin & Maint, internal only

**Review by george-lorch — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.7.34-37.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.34-37.rst]: Removed

File: doc/source/release-notes/Percona-Server-5.7.34-37.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.34-37.rst:21]: Added.

File: doc/source/release-notes/Percona-Server-5.7.34-37.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.7.34-37.rst]: PS-7651 removed.

**Review by george-lorch — APPROVED** [review]

**Pull request merged**